Wiki Updation #3 (#153)
NilPointer-Software/bdfd-wiki: src/premium/introduction.md

# Introduction
*Bot Designer for Discord*'s premium points system allows you to support the developers financially and gain some sweet perks!

### Purchasing Premium Points
1. Open BDFD app and click on profile icon.

    ![InShot_20220809_185729644](https://user-images.githubusercontent.com/95774950/183664366-78e3481f-656b-43b3-b334-4f4d00138fe3.jpg)

2. Then, click *"Buy premium points"*.

    ![Screenshot_20220809_190558](https://user-images.githubusercontent.com/95774950/183664494-1672d304-5250-4a6a-9bb3-60d2fbb960bc.png)

3. Choose your preferable package and confirm your purchase.

    ![Screenshot_20220809_114500](https://user-images.githubusercontent.com/95774950/183578062-f8b1074b-4462-4630-ae76-70ec670a97a6.png)

### Redeeming Premium

> After successful purchasing of premium point(s), here's how to redeem it :

Reviewer: Should be `purchase of premium points` and `redeem them`

```suggestion
After a successful purchase of premium points, here's how to redeem them :
```

1. Select your bot in app homepage and click *"Add premium time"* button in dashboard tab.

    ![Screenshot_20220809_112223](https://user-images.githubusercontent.com/95774950/183578142-e93ed985-9ea7-4b18-8d8b-3f29d073ad9f.png)

> 2. Select the amount of point(s) you want to add to the bot, then confirm.

Reviewer: Drop the `()`, and should be `you want to spend,`

```suggestion
2. Select the amount of points you want to spend, then confirm.
```

     ![IMG_20220809_152453](https://user-images.githubusercontent.com/95774950/183664582-2f2bbb33-819b-49c9-ab48-a4ec6500cd1e.jpg)

    > 📝 One premium point equals one week of premium hosting.

### Premium Perks
#### Main
- `$getServerVar[]` in triggers [(custom prefixes)](./customPrefixes.md).
- Awaited reactions.
- Access to [`$messageContains[]`](./messageContains.md) and [`$alwaysReply`](./alwaysReply.md) callbacks.
- Custom images.

> - Access to [`$ignoreTriggerCase`](./ignoreTriggerCase.md) and [`$sendNotification`](./sendNotification.md) function.

Reviewer: Should be `functions`

```suggestion
- Access to [`$ignoreTriggerCase`](./ignoreTriggerCase.md) and [`$sendNotification`](./sendNotification.md) functions.
```
